### DISCUSSION TITLE
마당이 올려준 evn 에 추가한 내용은 왜 추가한건가요...?! evn이란...?

### DISCUSSION THREAD

**Discussion by parkjooyoung0608** [EasyAndBeauty/DailyReceipt-FE-WEB#67] [Q&A]
REACT_APP_DAILY_RECEIPT_API_BASE_URL=http://3.36.239.183:8080
REACT_APP_KAUTH_REDIRECT_URI=http://localhost:3000/auth/kakao/callback
REACT_APP_KAKAO_BASE_URL =https://kauth.kakao.com/oauth/authorize

**Comment by IGhost-P** [comment]
env는 웹,앱 개발을 하다보면 포트, DB관련 정보, API_KEY등.. 개발자 혼자서 또는 팀만 알아야 하는 값 즉, git, 오픈소스에 올리면 안되는 값들이 있습니다.
로컬에서 작업을 할때는 자유롭게 이용하다가 github에 올릴때는 올리지 않게 하는식으로 이용됩니다!

**Comment by mdgarden** [comment]
* 클라이언트 ID는 공개되어도 상관없는 값이긴하지만 그래도 혹시 몰라 삭제했습니다.

API와 통신할 때, localhost:3000으로 시험삼아 로컬에서 연결하다가 실제로 프로덕션환경에 들어가게 되면 localhost:3000은 사용할 수 없는 주소가 됩니다.
이 때, 매번 로컬호스트 주소를 변경해주기 번거로우므로, REACT_APP_API_BASE_URL같은 식으로 환경변수를 지정하고 로컬이면 로컬 변수가, 실제 프로덕션 환경이면 프로덕션의 변수값을 받아오도록 지정한 것입니다.
